polynote-frontend/polynote/ui/input/hotkeys.ts

```diff
@@ -59,3 +67,21 @@ function keybindingToString(simpleKeybinding: Keybinding): string {
     }
     return keys.join("+")
 }
+
+export function keybindingFromString(keybindingStr: string, ): number {
```

Comment:
Now that we have some nontrivial logic (and errors being thrown) here, consider adding a test for this.